internal/ethapi/api.go

```diff
@@ -141,8 +141,8 @@ func (s *EthereumAPI) Syncing() (interface{}, error) {
 	return false, nil
 }
 
-func (s *BlockChainAPI) GetChainConfig(ctx context.Context) *params.ChainConfigWithUpgradesJSON {
-	return s.b.ChainConfig().ToWithUpgradesJSON()
+func (s *BlockChainAPI) GetChainConfig(ctx context.Context) *params.ChainConfig {
```

Comment:
If we think it's more convenient to return this without Upgrades, we should add a new API method for `GetUpgrades`. Otherwise we should revert this change and use  `ChainConfigWithUpgradesJSON` in both client and API function. 
cc @aaronbuchwald